README.rst

```diff
@@ -85,9 +85,14 @@ For support installing or managing Synapse, please join |room|_ (from a matrix.o
 account if necessary) and ask questions there. We do not use GitHub issues for
 support requests, only for bug reports and feature requests.
 
+Synapse's documentation is `nicely rendered on GitHub Pages <https://matrix-org.github.io/synapse>`_,
```

Comment:
Oh top of this, could we also add a badge to the file's header? e.g. https://img.shields.io/badge/-documentation-success
Personally that's where I look first for a link to a project's documentation.

Reply:
Good idea. I couldn't see one of these for documentation but didn't realise you could hand-craft your own.

Reply:
Yeah shields.io is a pretty slick resource :)